### ISSUE TITLE
Fix weak contrast ratio on `a:any-link` elements when `:target` pseudo-class detected

### ISSUE THREAD

**Issue by czerkies** [czerkies/romanczerkies.31#142]
- [x] Add alpha custom properties in `prefer-color-scheme: dark` to enhance maintainability of colors
- [x] https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#:~:text=Understanding%20SC%201.4.,(Minimum)%20(Level%20AA)&text=The%20visual%20presentation%20of%20text,Large%20Text)
- [x] Decrase alpha on secondary text color to enhance contrast ratio
- [x] Increase alpha custom properties on text colors to enhance dark mode legibility

**Comment by czerkies** [comment]
`--lbl-*-lph` Custom Properties have to be modified in `light` and `dark` mode

**Comment by czerkies** [comment]
Background missing contrast in `prefer-color-scheme: dark`.
